cost/currency_conversion/README.md

```diff
@@ -0,0 +1,41 @@
+# Currency Conversion
+
+## What it does
+
+This Policy creates an adjustment rule that converts the currency of the cost associated with the Cloud Vendor of choice. It utilizes xe.com to retrieve the latest exchange rates.
+
+## Prerequisites
+
+This Policy Template uses [Credentials](https://docs.flexera.com/flexera/EN/Automation/ManagingCredentialsExternal.htm) for authenticating to datasources -- in order to apply this policy you must have a Credential registered in the system that is compatible with this policy. If there are no Credentials listed when you apply the policy, please contact your Flexera Org Admin and ask them to register a Credential that is compatible with this policy. The information below should be consulted when creating the credential(s).
+
+- [**Flexera Credential**](https://docs.flexera.com/flexera/EN/Automation/ProviderCredentials.htm) (*provider=flexera*) which has the following roles:
+  - `enterprise_manager`
```

Comment:
Can someone help me identify what the correct role is to put here for 'Manage Organization'? This policy requires the ability to edit adjustments, which I believe requires 'Manage Organization' role.

Reply:
@bryankaraffa - still looking to merge this one. Not sure if you able to help with my above query around the correct permission/role required for editing adjustments?